### DISCUSSION TITLE
Not able to request native coins

### DISCUSSION THREAD

**Discussion by Pravveen38** [Cyfrin/foundry-full-course-cu#2908] [General]
![image](https://github.com/user-attachments/assets/65c06d66-db09-4a4c-b4c3-dc9c979b560a)
![Screenshot 2024-11-23 102320](https://github.com/user-attachments/assets/ae18e0e1-84bb-4ff0-8cc2-1d15042c0292)

**Comment by heheboii11** [comment]
may be you can use 
```https://cloud.google.com/application/web3/faucet/ethereum/sepolia```

**Reply by Pravveen38** [reply]
Thank you!!

**Comment by Pravveen38** [comment]
I used Google cloud web3. I successfully done my first transaction.

**Reply by heheboii11** [reply]
Congrats mate!! Keep on grinding!!
